cylc/flow/scheduler.py

```diff
@@ -711,6 +672,8 @@ def _load_pool_from_point(self):
 
     def _load_pool_from_db(self):
         """Load task pool from DB, for a restart."""
+        LOG.info("Restart from previous run")
+
```

Comment:
Is this needed? It already logs the restart number
```
INFO - Run: (re)start=4 log=1
```

Reply:
Nope, you're right. I missed that yesterday because all the reloaded task action timer stuff appears first.  Removing...